From: David Faure <faure@kde.org>
Date: Wed, 12 Aug 2015 19:47:19 +0000
Subject: Rename dolphin to dolphin4 and hide its .desktop file.
X-Git-Tag: v15.08.0
X-Git-Url: http://quickgit.kde.org/?p=kde-baseapps.git&a=commitdiff&h=bce5f70b75d4270e8902cffab4e1453f1104b8f4
---
Rename dolphin to dolphin4 and hide its .desktop file.

This way it doesn't appear to Applications/15.08 users, but
it's available to people compiling the latest-kde4 branch-group
from git (i.e. me, basically).

Disabled installation of files that conflict with dolphin.git
(appdata, .kcfg files, etc.); rename libdolphinprivate to libdolphinprivate4.

Anyhow the main point is to keep providing the part for Konqueror.

CCMAIL: emmanuelpescosta099@gmail.com
---


--- a/dolphin/src/CMakeLists.txt
+++ b/dolphin/src/CMakeLists.txt
@@ -346,15 +346,18 @@
 
 ########### install files ###############
 
-install( PROGRAMS dolphin.desktop DESTINATION ${XDG_APPS_INSTALL_DIR} )
-install( FILES settings/dolphin_directoryviewpropertysettings.kcfg
-               settings/dolphin_generalsettings.kcfg
-               settings/dolphin_compactmodesettings.kcfg
-               settings/dolphin_iconsmodesettings.kcfg
-               settings/dolphin_detailsmodesettings.kcfg
-               settings/dolphin_versioncontrolsettings.kcfg
-         DESTINATION ${KCFG_INSTALL_DIR} )
+# Disabled since Applications/15.08 contains kf5-based dolphin from dolphin.git
+#install( PROGRAMS dolphin.desktop DESTINATION ${XDG_APPS_INSTALL_DIR} )
+
+#install( FILES settings/dolphin_directoryviewpropertysettings.kcfg
+#settings/dolphin_generalsettings.kcfg
+#settings/dolphin_compactmodesettings.kcfg
+#settings/dolphin_iconsmodesettings.kcfg
+#settings/dolphin_detailsmodesettings.kcfg
+#settings/dolphin_versioncontrolsettings.kcfg
+#DESTINATION ${KCFG_INSTALL_DIR} )
 install( FILES dolphinui.rc DESTINATION ${DATA_INSTALL_DIR}/dolphin )
+#install( FILES dolphin.appdata.xml DESTINATION ${SHARE_INSTALL_PREFIX}/appdata )
 install( FILES search/filenamesearch.protocol DESTINATION ${SERVICES_INSTALL_DIR} )
 install( FILES settings/kcm/kcmdolphinviewmodes.desktop DESTINATION
 ${SERVICES_INSTALL_DIR} )
@@ -362,6 +365,6 @@
 install( FILES settings/kcm/kcmdolphinservices.desktop DESTINATION ${SERVICES_INSTALL_DIR} )
 install( FILES settings/kcm/kcmdolphingeneral.desktop DESTINATION ${SERVICES_INSTALL_DIR} )
 install( FILES settings/services/servicemenu.knsrc DESTINATION ${CONFIG_INSTALL_DIR} )
-install( PROGRAMS settings/services/servicemenuinstallation DESTINATION ${BIN_INSTALL_DIR} )
-install( PROGRAMS settings/services/servicemenudeinstallation DESTINATION ${BIN_INSTALL_DIR} )
+#install( PROGRAMS settings/services/servicemenuinstallation DESTINATION ${BIN_INSTALL_DIR} )
+#install( PROGRAMS settings/services/servicemenudeinstallation DESTINATION ${BIN_INSTALL_DIR} )
 
